RELEASES.md

```diff
@@ -138,13 +138,15 @@ for the list of actively tested versions. Kubernetes only supports n-3 minor
 release versions and containerd will ensure there is always a supported version
 of containerd for every supported version of Kubernetes.
 
-| Kubernetes Version | containerd Version | CRI Version     |
-|--------------------|--------------------|-----------------|
-| 1.24               | 1.7.0+, 1.6.4+     | v1, v1alpha2    |
-| 1.25               | 1.7.0+, 1.6.4+     | v1, v1alpha2 ** |
-| 1.26               | 1.7.0+, 1.6.15+    | v1              |
-| 1.27               | 1.7.0+, 1.6.15+    | v1              |
-| 1.28               | 1.7.0+, 1.6.15+    | v1              |
+| Kubernetes Version | containerd Versions           | CRI Version     |
+|--------------------|-------------------------------|-----------------|
+| 1.24               | 1.7.0+, 1.6.4+                | v1, v1alpha2    |
+| 1.25               | 1.7.0+, 1.6.4+                | v1, v1alpha2 ** |
+| 1.26               | 1.7.0+, 1.6.15+               | v1              |
+| 1.27               | 1.7.0+, 1.6.15+               | v1              |
+| 1.28               | 1.7.0+, 1.6.15+               | v1              |
+| 1.29               | 1.7.11+, 1.6.27+              | v1              |
+| 1.30(wip)          | 2.0(wip), 1.7.13+, 1.6.28+    | v1              |
```

Comment:
Probably we can just drop "wip", assuming that we can release containerd v2.0 before Kubernetes v1.30.

Reply:
nod .. the k8s release team was looking for some information regarding how we'll be aligning with their 1.30 release schedule

Reply:
would rather say wip until the release is finalized

Reply:
agree